_im-plugin/ism/policies.md

```diff
@@ -162,6 +163,46 @@ Parameter | Description | Type | Required
 
 For information about setting replicas, see [Primary and replica shards]({{site.url}}{{site.baseurl}}/opensearch#primary-and-replica-shards).
 
+### shrink
+
+Allows you to reduce the number of primary shards in your indexes. With this action, you can specify:
+
+- the number of primary shards that the target index should contain.
+- A max shard size for the primary shards in the target index.
+- Specify a percentage to shrink the number of primary shards in the target index.
+
+```json
+"shrink": {
+    "num_new_shards": {
+        "type": "integer"
+    },
+    "max_shard_size": {
+        "type": "keyword"
+    },
+    "percentage_of_source_shards": {
+        "type": "double"
+    },
+    "target_index_suffix": {
```

Comment:
In a recent change, this target_index_suffix is now "target_index_name_template". I updated the original backend PR with the change if you want to see some more documentation on that. This new field will have input that is formatted the exact same way as the notification action field 'message_template', so you can just copy what it notes there for consistency https://opensearch.org/docs/latest/im-plugin/ism/policies/#notification